velox/functions/sparksql/JsonFunctions.h

```diff
@@ -0,0 +1,209 @@
+/*
```

Comment:
Perhaps rename this header as 'GetJsonObject.h' because it is better to separate the implementation of each function.